CHANGELOG.md

```diff
@@ -0,0 +1,25 @@
+# Change Log
```

Comment:
looks like `changelog` is one word (refer https://en.wikipedia.org/wiki/Changelog).